src/test/java/org/cactoos/scalar/LengthOfTest.java

```diff
@@ -185,4 +188,28 @@ public void lengthOfEmptyIterator() {
             new IsEqual<>(0)
         ).affirm();
     }
+
+    @Test
+    public void lengthOfEmptyText() {
+        new Assertion<>(
+            "Must calculate length of empty string",
+            new LengthOf(new TextOf("")).intValue(),
+            new IsEqual<>(0)
+        ).affirm();
+    }
+
+    @Test
+    public void lengthOfText() {
+        final Number num = new LengthOf(new TextOf("abcd"));
```

Comment:
@andreoss can we add a test with some utf8 characters that does not translate directly to one byte? :)

Reply:
@victornoel You're suggestging that `LengthOf` should return a character-length for `Text`, not a byte-length? It seems inconsistent with how `Input` is handled.

Reply:
@andreoss exactly, that was a sneaky way of raising this problem in your solution, sorry about that ^^ We need to thus change the design to support both cases.

One possible solution is to have two classes (maybe as private inner class for now) for `Input` and `Text` and then have `LengthOf` delegate to them.

Reply:
@victornoel Done. It calls `asString().length()` for `Text` now.

Reply:
@andreoss hehe this was much simpler than what I had in mind :) thx